src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralServiceUnitTest.kt

```diff
@@ -44,13 +44,13 @@ class ReferralServiceUnitTest {
     val cancellationComments = "comment"
```

Comment:
As above

Reply:
changed the params, left the vars for now